### DISCUSSION TITLE
Bad S.O Signature in CentOS VM

### DISCUSSION THREAD

**Discussion by ggranata14** [Security-Onion-Solutions/securityonion#6884] [Unsupported Versions] [answered]
I have created a CentOS7 Virtualbox VM. It is set up with 2 NICs, one NAT in 10.0.2.x network and the second as an Internal Network in the 192.168.156.x network. 
I followed the following instructions for a CentOS system: 
https://docs.securityonion.net/en/2.3/installation.html#:~:text=If%20you%20want%20to%20install%20Security%20Onion%20using,the%20prompts%20to%20complete%20the%20installation%20and%20reboot.
I have installed git and was able to clone the repo successfully. However, when I try to implement the "sudo bash so-setup-network" command after trying 3 times, each time it would not get past the so-kibana install. After looking further into this, I discovered the following error in the setup log file:

Digest: sha256:e166e30fc1880a1dcdb24ca83822638ab4e31f4984ac60bb2a2e19fa269c14be
Status: Downloaded newer image for ghcr.io/security-onion-solutions/so-kibana:2.3.91
ghcr.io/security-onion-solutions/so-kibana:2.3.91 (0)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current    Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M100   543  100   543    0     0   1009      0 --:--:-- --:--:-- --:--:--  1011
There is a problem downloading the so-kibana:2.3.91 image. Details:
gpg: Signature made Mon 20 Dec 2021 10:04:08 AM EST using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.

It is telling me that I have a Bad Signature which is preventing me from getting past setup. Not sure how to proceed further to get a good signature or if there is another underlying issue causing this.

**Comment by dougburks** [comment, marked as answer]
> There is a problem downloading the so-kibana:2.3.91 image. Details:
> gpg: Signature made Mon 20 Dec 2021 10:04:08 AM EST using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC [info@securityonionsolutions.com](mailto:info@securityonionsolutions.com)"

This usually means that there is a firewall or some other device in your network path that is interfering with the download.

If you haven't already, you might try using our Security Onion ISO image as it's based on CentOS 7 and is our recommended installation method:
https://docs.securityonion.net/en/2.3/installation.html#installation-using-security-onion-iso-image
